v1.1/array.md

```diff
@@ -0,0 +1,64 @@
+---
+title: ARRAYS
+summary: The ARRAYS data type stores 
+toc: false
+---
+
+The `ARRAY` data type creates one-dimensional, 1-indexed, homogenous arrays of any non-array [data type](data-types.html).
+
+<div id="toc"></div>
+
+{{site.data.alerts.callout_info}} CockroachDB does not support nested arrays, indexing on arrays, and ordering by arrays.{{site.data.alerts.end}}
```

Comment:
This is fine, but just as a heads up it's caused some confusion during development that the word "indexing" here has two meanings - creating a database index (the meaning you're referring to here) and also indexing *into* an array (`SELECT (ARRAY[1,2,3])[2]`). I don't know if that's a concern for you but I vaguely suspect it could be a source of confusion, particularly among users new to databases.